x-pack/plugin/logsdb/src/yamlRestTest/resources/rest-api-spec/test/40_source_mode_setting.yml

```diff
@@ -590,7 +562,7 @@ create an index with logsdb index mode and disabled source:
               mapping.source.mode: disabled
 
   - match: { error.type: "mapper_parsing_exception" }
-  - match: { error.reason: "Failed to parse mapping: _source can not be disabled in index using [logsdb] index mode" }
+  - match: { error.reason: "Failed to parse mapping: _source cannot be disabled in index using [logsdb] index mode" }
```

Comment:
This is necessary because the error message introduced by the setting uses `cannot` instead of `can not`